tests/integration/bond_test.py

```diff
@@ -369,3 +370,17 @@ def test_bond_with_empty_ipv6_static_address(eth1_up):
         assertlib.assert_state(bond_state)
 
     assertlib.assert_absent('bond99')
+
+
+def test_create_vlan_over_a_bond_slave(bond99_with_slave):
+    vlan_bond_slave_iface = bond99_with_slave[Interface.KEY][0][
+        Bond.CONFIG_SUBTREE
+    ][Bond.SLAVES][0]
+    vlan_id = 102
+    vlan_iface_name = '{}.{}'.format(vlan_bond_slave_iface, vlan_id)
+    with bond_interface(name='bond99', slaves=['eth2']) as bond_state:
+        assertlib.assert_state(bond_state)
```

Comment:
Please move this line to the end of this function, so that we know the VLAN deletion does not impact bond.